### ISSUE TITLE
Release 3.0.0-rc.1 has broken WCT@5 on Typescript 2.1.x applications

### ISSUE THREAD

**Issue by owendismuke** [PolymerElements/test-fixture#45]
### Description
Polymer/web-component-tester#498
I know this is a fault in WCT, but I'm placing a reference here in case anyone else has to travel the rabbit hole I went through last night.

The issue is two-fold: 
1. WCT@5 does not reference a version of test-fixture in package.json (but they do reference a version in bower.json). This allows for exactly this issue to happen.
2. Adding dependency to WCT@6 has caused WCT@5 to fail on machines with less than Typescript 2.2.x. 

In an app using Typescript 2.1.6, I receive the below errors on a build. The errors are most likely caused by a dependency of WCT@6. However, I did notice that variant 1 has a dependency on WCT@4 - WCT@4.9x and variant 2 has a dependency on WCT@6+. Shouldn't variant 2 have a dep on WCT@5 - WCT@5.9x, and variant 3 start the dependency on WCT@6?

### Expected outcome
The app builds without an issue because I specified a specific version of WCT

### Actual outcome
```
Error: Build:']' expected.
Error: Build:';' expected.
Error: Build:Declaration or statement expected.
Error: Build:']' expected.
Error: Build:')' expected.
Error: Build:Declaration or statement expected.
Error: Build:Declaration or statement expected.
Error: Build:Declaration or statement expected.
```

### Steps to reproduce
1. Add WCT@5 to a project
2. Limit that project to typescript@~2.1.6
3. Write tests referencing `/components/web-component-tester/browsers.js`
4. Require WCT into gulp
5. Build project.

**Comment by jorgegarciafon** [comment]
Same issue here but caused by collission with `@types/jasmine`:
```
ERROR in [default] /src/node_modules/@types/jasmine/index.d.ts:9:17 
Duplicate identifier 'describe'.

ERROR in [default] /src/node_modules/@types/jasmine/index.d.ts:11:17 
Duplicate identifier 'xdescribe'.

ERROR in [default] /src/node_modules/@types/jasmine/index.d.ts:13:17 
Duplicate identifier 'it'.

ERROR in [default] /src/node_modules/@types/jasmine/index.d.ts:15:17 
Duplicate identifier 'xit'.

ERROR in [default] /src/node_modules/@types/mocha/index.d.ts:33:12 
Duplicate identifier 'describe'.

ERROR in [default] /src/node_modules/@types/mocha/index.d.ts:34:12 
Duplicate identifier 'xdescribe'.

ERROR in [default] /src/node_modules/@types/mocha/index.d.ts:39:12 
Duplicate identifier 'it'.

ERROR in [default] /src/node_modules/@types/mocha/index.d.ts:40:12 
Duplicate identifier 'xit'.
```

**Comment by DanielGarciaMandillo** [comment]
+1

**Comment by owendismuke** [comment]
I was able to fix this by using cloned repos:
[My WCT](https://github.com/owendismuke/web-component-tester/tree/fix-test-fixture-dep) & [My test-fixture](https://github.com/owendismuke/test-fixture/tree/fix-dependencies)

**Comment by owendismuke** [comment]
Fixed by #48
